modules/apps/company/company-test/src/testIntegration/java/com/liferay/company/service/test/CompanyLocalServiceTest.java

```diff
@@ -187,6 +194,37 @@ public void testAddAndDeleteCompany() throws Exception {
 		}
 	}
 
+	@Test
+	public void testAddAndDeleteCompanyWithAllLocalesEnabled()
+		throws Exception {
+
+		PropsValues.LOCALES_ENABLED = PropsUtil.getArray(PropsKeys.LOCALES);
```

Comment:
@Kvale1733 Make sure to set back PropsValues.LOCALES_ENABLED back to what it was originally otherwise this can break other tests, esp Locale ones